### ISSUE TITLE
No communication between containers in different hosts.

### ISSUE THREAD

**Issue by galal-hussein** [rancher/rancher#1183]
I 've been trying to use Ansible with Rancher, i created a MySQL container on one host and Wordpress container on another both with RANCHER_NETWORK=true, the wordpress container tries to connect to the mysql container immediately after creation which caused the the communication between the two hosts to stop.

Steps to reproduce:

1- set up a rancher server with two registered hosts
2- use these tasks to deploy both containers:
on node1
```
- name: Create a mysql docker container
  docker:
      name: mysql
      image: mysql:5.5
      detach: True
      env: RANCHER_NETWORK=true,
           MYSQL_ROOT_PASSWORD="secret"

- name: Wait a few minutes for the IPs to be set to the container
  wait_for: timeout=120

- name: Fetch the MySQL Container IP
  shell: docker exec mysql ip -o -4 addr list eth0 | awk '{print $4}' | cut -d/ -f1 |  sed -n 2p
  register: mysql_sec_ip
```
on node2
```
- name: Create a wordpress docker container
  docker:
      name: wordpress
      image: wordpress:latest
      detach: True
      ports:
      - 80:80
      env: RANCHER_NETWORK=true,
           WORDPRESS_DB_HOST={{ mysql_sec_ip.stdout }}:3306,
           WORDPRESS_DB_PASSWORD="secret",
 ```

expected results: wordpress up and running on port 80

results: no communication between containers on different hosts.

to work around this problem i had to create a dummy container before wordpress to give a time for network agent to start and set up the network:
```
- name: Create dummy container
  docker:
      name: busybox
      image: busybox
      env: RANCHER_NETWORK=true 

- name: wait for 20 sec
  wait_for: timeout=20
```

**Comment by will-chan** [comment]
This sounds very similar to the issue with #486.  It is something we are aware of and will be fixing soon.

**Comment by deniseschannon** [comment]
@galal-hussein We've just released v0.56.0 and replaced racoon with charon for our ipsec network. 

As this is similar to #486 and may be fixed, I'm closing. Please re-open a new issue if you see this again.
